src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -113,14 +118,6 @@ internal T CreateLoadedTypeItem<T>(NodeSearchElement element) where T: LoadedTyp
                         : string.Empty
             };
 
-            //If the node search element is part of a package, then we need to prefix pkg:// for it
```

Comment:
This is not needed, this case is already handled in GetFullyQualifiedName() method.